Files/MultilingualResources/Files.hu-HU.xlf

```diff
@@ -2399,43 +2399,43 @@
         </trans-unit>
         <trans-unit id="HorizontalMultitaskingControlCloseTabsToTheRight" translate="yes" xml:space="preserve">
           <source>Close tabs to the right</source>
-          <target state="new">Close tabs to the right</target>
+          <target state="translated">Jobbra lévő lapok bezárása</target>
         </trans-unit>
         <trans-unit id="StatusDeletionFailed" translate="yes" xml:space="preserve">
           <source>Deletion Failed</source>
-          <target state="new">Deletion Failed</target>
+          <target state="translated">Sikertelen törlés</target>
         </trans-unit>
         <trans-unit id="StatusUnknownError" translate="yes" xml:space="preserve">
-          <source>An unknown error has occurred.</source>
-          <target state="new">An unknown error has occurred.</target>
+          <source>Egy ismeretlen hiba történt.</source>
```

Comment:
Looks like something went wrong here, can you check? Source string should be kept.

Reply:
Fixed